honeybadger-io/honeybadger-js
# chore(webpack): set up basic example project to test plugin against

## Status
**READY**

## Description
In working on https://github.com/honeybadger-io/honeybadger-js/issues/1013, I found I needed a basic project to test sourcemap upload and deploy notifications. I decided to set up the project first so I could see the behavior on the existing package before making changes. 

## Steps to Test or Reproduce

```bash
> cd packages/webpack/example
> npm i
> npm run build
```
You should see 
```
Uploaded main.js.map to Honeybadger API
Successfully sent deploy notification to Honeybadger API.
```
